File: add/codesnippet/CPP/p-system.windows.forms.l_39_1.cpp
private:
   void SizeMyListBox()
   {
      // Add items to the ListBox.
      for ( int x = 0; x < 20; x++ )
      {
         listBox1->Items->Add( String::Format( "Item {0}", x ) );
      }
      listBox1->Height = listBox1->PreferredHeight;
   }

File: add/codesnippet/CSharp/p-system.windows.forms.l_39_1.cs
      private void SizeMyListBox()
      {
         // Add items to the ListBox.
         for(int x = 0; x < 20; x++)
         {
            listBox1.Items.Add("Item " + x.ToString());
         }
         // Set the height of the ListBox to the preferred height to display all items.
         listBox1.Height = listBox1.PreferredHeight;
      }